src/main/java/core/basesyntax/Drawable.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Drawable {
+    String draw();
```

Comment:
According to the checklist, methods in interfaces that represent an action should be named as verbs. The `draw` method could be better named to reflect this, such as `drawShape` or `render`.